vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -103,15 +107,19 @@ const char *ImageViewManager::GetName() const {
 }
 
 XamlView ImageViewManager::CreateViewCore(int64_t tag) {
-  return ReactImage::Create().as<winrt::Canvas>();
+  auto image = ReactImage::Create().as<winrt::Canvas>();
+  auto border = winrt::Border();
+  border.Child(image);
```

Comment:
Yesterday in our discussion, I had a question about whether or not the image is supposed to draw underneath the border (as opposed to being inset).  Did you find out how this behaves on other platforms?

Reply:
Had a discussion with Keith offline, we decided to switch image from Canvas to Grid so we can set border properties on it and still just have 1 element for image in the tree. That means we can't support background property on Image for now, that requires at least three elements, and implementing TransferProperties, Reparent etc. Out of scope for this bug fix but I will open a new issue to track the background for image for fix at later time.

---
In reply to: [357410515](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r357410515) [](ancestors = 357410515)